README.md

```diff
@@ -1,4 +1,4 @@
-# fusion-plugin-universal-events
+  # fusion-plugin-universal-events
 
 [![Build status](https://badge.buildkite.com/de4e30ddb9d019f5a8e3a2519bc0a5cccab25247809cd10c99.svg?branch=master)](https://buildkite.com/uberopensource/fusion-plugin-universal-events)
 
```

Comment:
There should be a section talking about why we didn't use an existing event emitter library and how the API and semantics of our library differs from other event emitters.

If this library is not a suitable implementation for a hypothetical `EventEmitterToken` interface, we should also mention that.